hvcc/generators/ir2c/static/HvControlExpr.h

```diff
@@ -41,7 +41,8 @@ float expr_modf(float mod);
 
 float expr_imodf(float mod);
 
-float expr_if(float eval, float trueValue, float falseValue);
+//float expr_if(float eval, float trueValue, float falseValue);
+#define expr_if(q, a, b) ((q)?(a):(b))
```

Comment:
I think you have the wrong code in this PR?

Reply:
oops, sorry, no idea what happened. Will close and open a new PR.

Reply:
should be fine now in this PR, corrected the respective branch

Reply:
In fact, with C99 accepted, these (i)modf functions should then be inline functions in HvControlExpr.h.